content/steps-to-become-a-teacher.md

```diff
@@ -66,22 +66,11 @@
 
 ## Become a qualified teacher in England
 
-Follow these steps if you want to become a qualified primary or secondary school teacher in England.
+Follow these steps if you want to become a qualified primary or secondary school teacher in England. You'll need to work towards Qualified Teacher Status (QTS). This can be achieved in different ways, including:
 
-If you’re interested in teaching in another area, take a look at our guidance on:
+* doing a postgraduate initial teacher training (ITT) course if you already have a degree
+* doing an undergraduate degree course which leads to QTS
 
-* [becoming a further education teacher](/further-education-teacher-training)
-* [becoming an early years teacher](/early-years-teaching-training)
+If you're interested in teaching a different age group, consider training to teach [early years](/early-years-teaching-training) or [further education](/further-education-teacher-training).
```

Comment:
I'm not sure 'different age group' works here - it's not clear what they're different to. Could it perhaps be more explicit and say "younger children or 16+" (obviously worded more nicely!)

Reply:
Agree - I'll push a change

Reply:
```suggestion
If you're not looking to teach primary or secondary, consider training to teach [early years](/early-years-teaching-training) or [further education](/further-education-teacher-training).
```

Reply:
Thanks @peteryates. I've added in a temporary fix.